pytorch_toolkit/object_detection/oteod/oteod/training/common.py

```diff
@@ -0,0 +1,126 @@
+# pylint: disable=W1203
+
+import json
+import logging
+import os
+import subprocess
+import sys
+import tempfile
+
+import torch
+import yaml
+from mmcv.utils import Config
+from oteod import MMDETECTION_TOOLS
+from oteod.misc import run_with_termination
+
+
+def train_internal(config, gpu_num, update_config):
+    training_info = {'training_gpu_num': 0}
+    if os.getenv('MASTER_ADDR') is not None and os.getenv('MASTER_PORT') is not None:
+        logging.info('Distributed training started ...')
+        run_with_termination(f'python {MMDETECTION_TOOLS}/train.py'
```

Comment:
This case has been brought by our colleagues from another team. I can't comment how they will use it, probably they know it better. They don't use dist_train.sh

Reply:
That's up to them. Though, log message says that this is distributed training, while it's obviously not. And comment about the number of training GPUs is still valid.

Reply:
"Distributed training on the Platform is achieved by integration with Kubeflow’s PyTorchJob. Because of that, we cannot rely on “dist_train.sh” script to start distributed training. However, it will be enough to execute train.py script with --launcher=pytorch argument in each training Pod on the Kubernetes cluster. All necessary env variables like MASTER_ADDR and MASTER_PORT will be there."

Their platform does not rely on our 'training_gpu_num'

Reply:
To be honest it would be great if you did not leave discussion on this and related stuff..

Reply:
Please put a comment, that distributed multi-GPU training is handled by Kubeflow’s PyTorchJob at a higher level.

Reply:
I guess it can be not only GPU training. So added "Distributed training is handled by Kubeflow’s PyTorchJob at a higher level."